sdk/openai/openai/test/public/utils/recordedClient.ts

```diff
@@ -31,6 +31,16 @@ const recorderStartOptions: RecorderStartOptions = {
         target: `\\.png?[^"]+`,
         value: ".png?sanitized",
       },
+      {
+        regex: true,
+        target: `resourceGroups/[\\w-]+/`,
+        value: "resourceGroups/openai-shared/",
+      },
+      {
+        regex: true,
+        target: `accounts/[\\w-]+/`,
+        value: "accounts/openai-shared/",
+      },
```

Comment:
We shouldn't need this anymore once the whisper PR gets merged.

Reply:
I'll remove this for this PR